projects/plugins/jetpack/tests/php/modules/masterbar/test-class-admin-menu.php

```diff
@@ -288,7 +288,6 @@ public function test_add_plugins_menu() {
 		static::$admin_menu->add_plugins_menu();
 
 		$this->assertSame( 'https://wordpress.com/plugins/' . static::$domain, $menu[65][2] );
-		$this->assertEmpty( $submenu['plugins.php'] );
```

Comment:
Instead of removing this assert, can we change it so we check there are no visible submenus?